R/client-connect.R

```diff
@@ -45,7 +45,7 @@ connectClient <- function(service, authInfo) {
         c("account_id", names(filters)),
         c(accountId, unname(filters))
       ), collapse = "&")
-      listRequest(service, authInfo, path, query, "applications")
+      listApplicationsRequest(service, authInfo, path, query, "applications")
```

Comment:
Any concerns about this bug also occuring for other clients?

Reply:
No. The Connect internal applications API does pagination differently from other APIs.